pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -789,10 +796,14 @@ func (c *Cluster) getActivateOSDInitContainer(configDir, namespace, osdID string
 			Type: &hostPathType,
 		},
 	}
-	volume := v1.Volume{
+	volume = append(volume, v1.Volume{
```

Comment:
You don't need to call `append` and initialize `volume` earlier, just do:


```go
	volume := []v1.Volume{
		{
			Name:         activateOSDVolumeName,
			VolumeSource: source,
		},
	}
```

Reply:
done.